modules/web/server/src/main/scala/navigate/web/server/http4s/WebServerLauncher.scala

```diff
@@ -60,14 +60,8 @@ object WebServerLauncher extends IOApp with LogInitialization {
   def configurationFile[F[_]: Sync]: F[FilePath] =
     baseDir[F].map(_.resolve("conf").resolve("app.conf"))
 
-  // Try to load config from the file and fall back to the common one in the class path
```

Comment:
We only deploy a single `app.conf` with navigate, there's no fallback config with defaults.